Source/WebCore/css/parser/CSSSelectorParser.cpp

```diff
@@ -312,7 +312,11 @@ std::unique_ptr<CSSParserSelector> CSSSelectorParser::consumeComplexSelector(CSS
     for (CSSParserSelector* simple = selector.get(); simple && !previousCompoundFlags; simple = simple->tagHistory())
         previousCompoundFlags = extractCompoundFlags(*simple, m_context.mode);
 
-    while (auto combinator = consumeCombinator(range)) {
+    while (true) {
+        auto combinator = consumeCombinator(range);
+        if (combinator == CSSSelector::RelationType::Subselector)
```

Comment:
This seems like the main functional change for this patch. Should it be referenced somehow in the Commit Log?

Reply:
It's not a functional change (hopefully 😅 ). But `combinator` is not implicitly convertible to `bool` now (it's kinda the point of type safe enum) so we need to do the comparison with 0 (with is RelationType::Subselector) explicitly.